tests/test_data_sources_nods.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import pytest
+
+
+@pytest.mark.skipif(
+    os.getenv("DATA_SDK") == "yes", reason="Testing failure when dependency is missing"
+)
+def test_data_sources_import_fails():  # noqa
```

Comment:
out of curiosity, what was the reasoning for moving this to a different file?

Reply:
also what is the `_nods` part in the filename referring to?

Reply:
this is a pre-work for testing purposes with tox to test for different environment values. ds :: dataset downloaded

Reply:
So, one set of test test if the dataset is downloaded, and one if not.

Reply:
so I better understand, why does it need to be moved to a different file to test for different environment values?

so is nods short for no data sets?

Reply:
yes.

Reply:
see tox.ini, other wise, it returns false warning.

Reply:
getting rid of a misleading warning sounds good

Reply:
It wasn't obvious to me what "nods" meant so I think it could make sense to rename to something like "test_no_data_sources.py"